pygmt/clib/session.py

```diff
@@ -1086,7 +1089,9 @@ def virtualfile_from_vectors(self, *vectors):
         for col, array in enumerate(arrays):
             self.put_vector(dataset, column=col, vector=array)
 
-        with self.open_virtual_file(family, geometry, "GMT_IN", dataset) as vfile:
+        with self.open_virtual_file(
+            family, geometry, "GMT_IN|GMT_IS_REFERENCE", dataset
```

Comment:
It's interesting that `virtualfile_from_grid()` function already uses `GMT_IN|GMT_IS_REFERENCE` but the `_vectors` one doesn't. Anyways, just a comment.